### ISSUE TITLE
"invalid cookie domain" when adding cookie to browser

### ISSUE THREAD

**Issue by jprealini** [MicrosoftEdge/EdgeWebDriver#9]
Hello.. I've been trying to solve this issue or find any documentation or reference to it, but no luck.
I am working on adding Edge to a Selenium framework which was only supporting Chrome, Firefox and IE. The framework was running on Selenium 3.4, so first I decided to upgrade Selenium to 4.1. Then also downloaded the msedgedriver.exe version for the browser version I have (100.0.1185.29).
The solution is built with C#, and the logic to add cookies is the same used for Chrome and Firefox (for which I don't have any issues)
When trying to add the same cookies to Edge, I get an error saying "invalid cookie domain", even though the cookie's domain is set to the exact same domain I am visiting for the test
Have you seen this issue before? Maybe I am missing something?
Thanks

**Comment by bwalderman** [comment]
@jprealini can you share a minimal test that reproduces this behavior?

**Comment by jprealini** [comment]
@bwalderman hi! so, I already solved this, but forgot to come back and report. The issue had to do with a mismatch between driver version and browser version. For some reason, Selenium was picking up an old driver (for edge 99) instead of the new driver I downloaded. So I had to make a cleanup, make sure to add keep only the latest driver, and problem was solved... Probably worth to add a note somewhere "IF YOU SEE THIS INVALID COOKIE DOMAIN ISSUE, MAKE SURE YOU HAVE THE RIGHT DRIVER LOADED"... 😆
